.github/styles/Vocab/HMSVocab/accept.txt

```diff
@@ -119,4 +119,5 @@ navbar
 upload_info
 rtmpDestinations
 hlsDestinations
-WebKit
\ No newline at end of file
+WebKit
+goswami
```

Comment:
where is this used?

Reply:
[changelog](https://100ms-docs-77qws43dq-100mslive.vercel.app/docs/android/v2/changelog/release-notes#200-alpha04-2021-06-04)

Reply:
please remove that line. it's not useful

Reply:
and this change to accept.txt

Reply:
sure